### PR TITLE
Use ENTRYPOINT instead of CMD for starting containers

### PR DESCRIPTION
**Pull request by r0man**
I'm working on my 2nd into-docker builder for sqitch [1] and noticed
that into-docker fails to build because the runner image dies. I
believe this is because the runner image I'm using has defined an
ENTRYPOINT and a CMD [2]. So, when the into-docker starts the builder
and the runner image by executing `tail -f /dev/null` I think on the
the runner image the command that gets executed is `/bin/sqitch tail
-f /dev/null`, which fails and the runner container exists.

I got it working by using the `:Entrypoint` instruction in
`invoke-run-container` instead of `:Cmd`.

Wdyt?

[1] https://sqitch.org/
[2] https://github.com/sqitchers/docker-sqitch/blob/main/Dockerfile#L67

### CODE REVIEW

File: src/into/docker/container.clj [hunk 1]
```diff
@@ -79,7 +79,7 @@
               :params {:name  container-name
                        :body {:Image full-name
                               :User  user
-                              :Cmd   ["tail" "-f" "/dev/null"]
+                              :Entrypoint ["tail" "-f" "/dev/null"]
```
Comment (xsc) [inline src/into/docker/container.clj]: That works, but just to be safe (and very explicit), I'd suggest to use:

```suggestion
                              :Entrypoint nil
                              :Cmd ["tail" "-f" "/dev/null"]
```

**Comment by xsc** [comment]
Thanks, @r0man ! And don't worry about the failing checks there, it seems like I'll have to modify some pipeline behaviour.

File: src/into/docker/container.clj (hunk 1 above)
Comment (r0man) [reply src/into/docker/container.clj]: Ah, yes, that is probably better.

**Review by xsc — APPROVED** [review]

**Comment by xsc** [comment]
Available in [`1.1.0-RC2`](https://github.com/into-docker/into-docker/releases/tag/v1.1.0-RC2).

**Pull request merged**